### ISSUE TITLE
Coq 8.8 raises anomalies when printing inductive type declarations (Anomaly~"File~"printing/ppcons...)

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#10222]
#### Description of the problem

```
$ cat bar.v
Inductive x := y.
$ for i in 8.7.0 8.8.0 8.9.0; do ~/.local64/coq/coq-$i/bin/coqc -v; ~/.local64/coq/coq-$i/bin/coqc -q -time ba
r.v; echo ---------; done
The Coq Proof Assistant, version 8.7.0 (December 2017)
compiled on Dec 22 2017 15:58:58 with OCaml 4.02.3
Chars 0 - 17 [Inductive~x~:=~~~~~y~:~_.] 0.001 secs (0.u,0.s)
---------
The Coq Proof Assistant, version 8.8.0 (January 2019)
compiled on Jan 28 2019 16:30:47 with OCaml 4.06.1
Chars 0 - 17 [Anomaly~"File~"printing/ppcons...] 0. secs (0.u,0.s)
---------
The Coq Proof Assistant, version 8.9.0 (April 2019)
compiled on Apr 9 2019 15:50:46 with OCaml 4.06.1
Chars 0 - 17 [Anomaly~"File~"printing/ppcons...] 0. secs (0.u,0.004s)
---------
```
#### Coq Version

8.8.0, 8.9.0

**Comment by JasonGross** [comment]
cc @ejgallego ?

**Comment by ejgallego** [comment]
This is fixed in 8.10

**Comment by JasonGross** [comment]
Which PR fixed it?
